docs/widgets/label.md

```diff
@@ -0,0 +1,33 @@
+# Label
+
+A widget which displays static text, but which can also contain more complex Rich renderables.
+
+- [ ] Focusable
+- [x] Container
```

Comment:
A Label can't contain children, which means it isn't a container. I think this is inherited from Static, which is also incorrect.

Reply:
Good catch, I'll fix `Static` and then sync this.

Reply:
> A Label can't contain children, which means it isn't a container. I think this is inherited from Static, which is also incorrect.

Actually... this isn't this isn't quite true is it? Especially not for static. A Static can comprise of other widgets. Consider [the stopwatch tutorial](https://textual.textualize.io/tutorial/#custom-widgets) for example.